index.html

```diff
@@ -236,33 +238,65 @@ <h2>Extensions to the <dfn data-cite=
       <section>
         <h2><dfn>getEntries()</dfn> method</h2>
         <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
-        "#filter-buffers-by-name-and-type"></a> algorithm with <var>buffers</var>
-        set to a list containing a <a>relevant performance entry buffer</a> for
-        each entry type supported by the user agent and whose
-        <a>availableFromeTimeline</a> boolean is set to true and <var>name</var>
-        and <var>type</var> set to <code>null</code>.</p>
+        "#filter-buffers-by-name-and-type"></a> algorithm with
+          <ol>
+            <li>
+              <var>buffers</var> set to a list containing
```

Comment:
I think it is better to split this out into a new algorithm which is similar to filter-buffers-by-name-and-type, except that it excludes entries for which availableFromTimeline is false. This would having this structure 3 times. What do you think?

Reply:
Done with Yoav's changes.